content/en/blog/hacktoberfest-2021/index.md

```diff
@@ -0,0 +1,119 @@
+{
+    "title": "Hacktoberfest 2021: Hack with us!",
+    "type": "blog",
+    "date": "2021-09-19T11:00:0+02:00",
+    "description": "We are again participating in the Hacktoberfest! Contribute and get a webcam slider, sticker set and maybe a free t-shirt.",
+    "featured_image": "hacktoberfest-2021",
+    "tags": [ "hacktoberfest", "digitalocean", "github", "open source", "pull request" ],
+    "authors": [ "malte","baltpeter" ],
+    "notices": "Photo adapted after: \"[maple leaves photo](https://unsplash.com/photos/Pu-en3ew8wY)\" by [Fey Marin](https://unsplash.com/@feymarin) ([Unsplash-Lizenz](https://unsplash.com/license))"
+}
+~~Party time!~~ October and therefore the Hacktoberfest are back! We are participating again and are looking forward to your contributions. Just like in the last years, we are happy to give away some goodies as well.
+
+{{< featuredImg alt="Written infront of maple leaves: \"Hacktoberfest 2021: Hack with us!\"" >}}
```

Comment:
Looks more like oak tbh. but that's how the picture is called 🤔

Reply:
Clearly that is not maple :DD

Reply:
Glad we're discussing the important stuff. I for one think that the leaf type _really_ doesn't matter too much here and we can just call them 'leaves'. :D